docs/docsite/rst/playbook_guide/playbooks_templating_now.rst

```diff
@@ -13,4 +13,7 @@ utc
   Specify ``True`` to get the current time in UTC. Defaults to ``False``.
 
 fmt
-  Accepts a `strftime <https://docs.python.org/3/library/datetime.html#strftime-strptime-behavior>`_ string that returns a formatted date time string.
\ No newline at end of file
+  Accepts a `strftime <https://docs.python.org/3/library/datetime.html#strftime-strptime-behavior>`_ string that returns a formatted date time string.
+
+Example
+  dtg: "Current time (UTC): {{ now(utc=true,fmt='%Y-%m-%d %H:%M:%S') }}"
```

Comment:
```suggestion
For example: ``dtg: "Current time (UTC): {{ now(utc=true,fmt='%Y-%m-%d %H:%M:%S') }}"``
```

Reply:
Looks good but I think it would be better to reformat a little so the example doesn't get treated as part of the definition list.

Here is how it appears in the rendered docs:

![image](https://github.com/ansible/ansible-documentation/assets/32749632/717e8caf-3ee2-433e-bbb1-8541459dae48)